client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -95,7 +98,7 @@ const Checkout = React.createClass( {
 		props = props || this.props;
 
 		analytics.tracks.recordEvent( 'calypso_checkout_page_view', {
-			saved_cards: props.cards.get().length,
+			saved_cards: props.cards.length,
```

Comment:
I wonder if we shouldn't record this event only when the list of cards have loaded from the server, otherwise we might just provide stale data to Tracks. This is how it currently works though.